Formula/osinfo-db-tools.rb

```diff
@@ -0,0 +1,27 @@
+class OsinfoDbTools < Formula
+  desc "Tools for managing the libosinfo database files"
+  homepage "https://libosinfo.org"
+  url "https://releases.pagure.org/libosinfo/osinfo-db-tools-1.1.0.tar.gz"
+  sha256 "a141cd2fc07c30d84801b5dbf6b11f2c2e708b0e81216277d052ac0b57fe546b"
+  depends_on "intltool" => :build
```

Comment:
Newline between `sha256` and `depends_on`

Reply:
done